docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -0,0 +1,325 @@
+These files are read during boot (and when `systemctl daemon-reload` is run) and generate
+corresponding regular systemd service unit files. Both system and user systemd units are supported.
+
+The podman generator reads the search paths above and reads files with the extensions `.container`
+and `.volume`, and for each file generates a similarly named `.service` file. These units can be started
+and managed with systemctl like any other systemd services.
+
+The podman files use the same format as [regular systemd unit files](https://www.freedesktop.org/software/systemd/man/systemd.syntax.html).
+Each file type has a custom section (for example `[Container]`) that is handled by podman, and all
+other sections will be passed on untouched. This means you can use any normal systemd configuration
+like dependencies or cgroup limits.
+
+### Enabling unit files
+
+The services created by podman are considered transient by systemd, which means they don't have the same
+persistence rules as regular units. In particular, it is not possible to "systemctl enable" them
+in order for them to become automatically enabled on the next boot.
+
+To compensate for this, the generator manually applies the `[Install]` section of the container definition
+unit files during generation, in the same way `systemctl enable` would do when run later.
+
+For example, to start a container on boot, you can add something like this to the file:
+
+```
+[Install]
+WantedBy=multi-user.target
+```
+
+Currently only the `Alias`, `WantedBy` and `RequiredBy` keys are supported.
+
+**NOTE:** If you want to express dependencies between containers you need to use the generated names
+of the service. In other words `WantedBy=other.service`, not `WantedBy=other.container`. The same is
+true for other kinds of dependencies too, like `After=other.service`.
+
+### Container units
+
+Container units are named with a `.container` extension and contain a `[Container] `section describing
+the container that should be run as a service. The resulting service file will contain a line like
+`ExecStart=podman run … image-name`, and most of the keys in this section control the command-line
+options passed to podman.  However, some options also affect details of how systemd is set up to run and
+interact with the container.
+
+By default the podman container will have the same name as the unit, but with a `systemd-` prefix.
+I.e. a `$name.container` file will create a `$name.service` unit and a `systemd-$name` podman container.
+
+There is only one required key, `Image` which defines the container image that should be run by the service.
+
+Supported keys in `Container` group are:
+
+#### `Image=`
+
+   The image to run in the container. This image must be locally installed for the service to work
+   when it is activated, because the generated service file will never try to download images.
+   It is recommended to use a fully qualified image name rather than a short name both for
```

Comment:
I think Quadlet should probably warn in the case where it is not fully qualified.

Reply:
Perhaps mention using the Digest for guaranteed execution of the required app.

Reply:
I will later add a warning abouit the non fully qualified case.